packages/api-server/src/helpers.ts

```diff
@@ -0,0 +1,27 @@
+import { getConfig } from '@redwoodjs/project-config'
+
+export function getAPIHost() {
```

Comment:
Personally I prefer not to upper-case all letters in Api when used in a camel case method name like this. But totally just personal preference, no new rule or anything like that
```suggestion
export function getApiHost() {
```

Reply:
Never mind. It's good 🙂 We're consistent here with both API and CLI in all places